packages/consumption/test/modules/requests/IncomingRequestsController.test.ts

```diff
@@ -954,6 +982,37 @@ describe("IncomingRequestsController", function () {
             await Then.theNumberOfReturnedRequestsIs(2);
         });
 
+        test("updates the expiration date if the Template expires before the Request", async function () {
+            const timestamp = CoreDate.utc();
+            const request = await Given.anIncomingRequestWith({
```

Comment:
Also @jkoenig134, why did you remove the expiration update stemming from the templates from the getters?

Reply:
it was useless as the expiresAt is written inside the received method

Reply:
this was meant for requests that already exist in the database before the runtime is updated

Reply:
IDK about those ones.

Reply:
no need to clutter the codebase for that

Reply:
and people then get to be surprised by a not expired request ending up being expired because of the template? Or do you think that the pair of expiration dates has been used sensibly so far?

Reply:
they don't think anything b/c the app will anyways call the canCreateRelationship usecase early enough and that will explode for them :) so ezpz